PocketKit/Sources/Sync/Collection/CollectionService.swift

```diff
@@ -8,37 +8,25 @@ import PocketGraph
 import SharedPocketKit
 
 protocol CollectionService {
-    func fetchCollection(by identifier: String) async throws -> CollectionModel
-}
-
-// TODO: Refine error handling
-public enum CollectionServiceError: Error {
-    case anError
+    func fetchCollection(by identifier: String) async throws
 }
 
 class APICollectionService: CollectionService {
     private let apollo: ApolloClientProtocol
+    private let space: Space
 
-    init(apollo: ApolloClientProtocol) {
+    init(apollo: ApolloClientProtocol, space: Space) {
         self.apollo = apollo
+        self.space = space
     }
 
-    func fetchCollection(by slug: String) async throws -> CollectionModel {
+    func fetchCollection(by slug: String) async throws {
         let query = GetCollectionBySlugQuery(slug: slug)
 
         guard let remote = try await apollo.fetch(query: query).data?.collection else {
             Log.capture(message: "Error loading collection")
-            throw CollectionServiceError.anError
+            return
```

Comment:
shall we add a `TODO` here to discuss how to handle errors?

Reply:
Errors should be captured, and depending on the criticallity shown to the user.